docker/docker-compose.yml

```diff
@@ -337,6 +337,7 @@ services:
       dockerfile: docker/test-php/Dockerfile
     image: test-php
     container_name: test-php
+    platform: linux/amd64
```

Comment:
so, is this necessary again now?

Reply:
thanks for catching this!  There are several unintended commits in this branch that I will rebase and delete, and then this should go away